src/components/common/PeopleAndRoles.vue

```diff
@@ -184,8 +184,8 @@
         <span>Add the Completing Party</span>
       </v-btn>
 
-      <!-- *** FUTURE: don't show Add a Person for short-form amalgamation -->
       <v-btn
+        v-if="getPeopleAndRolesResource.addPerson !== false"
```

Comment:
I decided on this logic so that I wouldn't have to update all the existing resource files.